### PR TITLE
README: add note about location for macOS

### PR DESCRIPTION
**Pull request by adidalal**
Just a small update to point out that on M1, the path will be different.

Thanks for putting this together!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,11 +125,11 @@ Many Linux distributions nowadays use [Systemd](https://en.wikipedia.org/wiki/Sy
    ```console
 	$ make PREFIX=/usr/local install-launchagent
    ```
-1. Fill out [configuration values](#2-configure-b2-credentials-locally) in `/usr/local/etc/restic`.
+1. Fill out [configuration values](#2-configure-b2-credentials-locally) in `$HOMEBREW_PREFIX/etc/restic`.
 1. [Initialize](#3-initialize-remote-repo) the remote repo.
 	Source the profile to make all needed configuration available to `restic(1)`. All commands after this assumes the profile is sourced in the current shell.
    ```console
-	$ source /usr/local/etc/restic/default.env.sh
+	$ source $HOMEBREW_PREFIX/etc/restic/default.env.sh
```
Comment (erikw) [inline README.md:132]: Ditto

```console
$ source $(brew --prefix)/etc/restic/default.env.sh
```

**Review by erikw — CHANGES_REQUESTED** [review]
Thank you for this PR! I was not aware of the change for Apple silicon Macs.

A few changes needed, see individual comments.

File: README.md [hunk 2]
```diff
@@ -125,11 +125,11 @@ Many Linux distributions nowadays use [Systemd](https://en.wikipedia.org/wiki/Sy
    ```console
 	$ make PREFIX=/usr/local install-launchagent
    ```
-1. Fill out [configuration values](#2-configure-b2-credentials-locally) in `/usr/local/etc/restic`.
+1. Fill out [configuration values](#2-configure-b2-credentials-locally) in `$HOMEBREW_PREFIX/etc/restic`.
```
[the remark this answers is not in the document]
Comment (adidalal) [reply README.md:128]: I use `eval "$(/opt/homebrew/bin/brew shellenv)"` in my profile, but same idea. Updated everywhere

**Comment by erikw** [comment]
Thanks!

**Pull request merged**